profile/README.md

```diff
@@ -1,41 +1,86 @@
+* [The Arcaflow engine](https://github.com/arcalot/arcaflow-engine), statically-compiled
+and platform-portable, written in Go
+* [Plugins](https://github.com/orgs/arcalot/repositories?q=arcaflow-plugin-), written in
+various languages and delivered as Linux containers or modules
+* [Workflow definitions](https://github.com/arcalot/arcaflow-workflows), written in
+YAML, tying together plugins and their data
 
+<br clear="left"/>
 <br clear="left"/>
 
-## Arcalog
-
-<img align="right" width="200px" style="padding-left: 2em;"  alt="Arcalog logo showing an ark floating on a blue scroll symbolizing the many logs it is scrolling through" src="https://github.com/arcalot/.github/raw/main/branding/arcalog.png">
-
-Arcalog can assist you with or automate your root cause analysis in CI or other log systems either as a standalone tool or by embedding it into your applications.
-
-It also provides additional tooling to download jobs from various log systems or add your own log files for analysis.
-
-<br clear="right"/>
+Arcaflow is highly-flexible and portable, helping you to build
+pipelines of actions via plugins. Plugin steps typically perform one action well, 
+creating or manipulating data that is returned in a machine-readable format. Data is
+validated according to schemas as it passes through the pipeline in order to clearly
+diagnose type mismatch problems early. Arcaflow runs on your laptop, a jump host, or in
+a CI system, requiring only the Arcaflow engine binary, a workflow definition in YAML,
+and a compatible container runtime.
+
+Arcaflow allows you to encapsulate and version-control expertise, making potentially
+very complex workflows easily portable among environments and automation systems. With
+an Arcaflow workflow, you can carefully craft a pipeline of actions that serves your
+direct needs and share that worflow virtually unchanged for others to run in different
+environments and CI/CD systems.
+
+An ever-growing catalog of
+[official plugins](https://github.com/orgs/arcalot/repositories?q=arcaflow-plugin-) are
+maintained within the Arcalot organization and are available as
+[versioned containers from Quay.io](https://quay.io/organization/arcalot). You can also
+build your own containerized plugins using the the Arcaflow SDK, available for
+[Python](https://arcalot.io/arcaflow/creating-plugins/python/) and
+[Golang](https://github.com/arcalot/arcaflow-plugin-sdk-go). We encourage you to
+contribute your plugins to the community, and you can start by adding them to the
+[plugins incubator](https://github.com/arcalot/arcaflow-plugins-incubator) repo via a
+pull request.
+
+![image](https://raw.githubusercontent.com/arcalot/arcaflow-engine/version-and-flags/arcaflow-basic-demo.gif)
+
+# Documentation
+
+We work hard to bring the documentation to the user, meaning that you should fine a lot
```

Comment:
I think we want users to "find" documentation rather than "fining" it (perhaps for imperfection?) 😆